tests/mock_crust/utils.rs

```diff
@@ -325,10 +325,18 @@ pub fn create_connected_nodes_with_cache(network: &Network,
     nodes
 }
 
-pub fn create_connected_nodes_with_cache_till_split(network: &Network) -> Vec<TestNode> {
+// This creates new nodes (all with `use_cache` set to `true`) until the specified number of unique
+// split events have happened.  `split_count` should be at least 1.  Note that the final shape of
+// the network is still random, e.g. if `split_count` is 5, then the network could comprise groups
```

Comment:
It took me a few shots to get this right, but I think 5 is correct :)  The events notify about the new prefix, so there would be `0`, `1`, `10`, `11`, then finally `110` or `111` takes us to the state in the first example.